test/unit/user/user.spec.js

```diff
@@ -9,8 +9,25 @@ import {
   successfulSignup,
   successfulLogin,
   nonExistingEmail,
-  incorrectPassword
+  incorrectPassword,resetPasswordSuccessful,
+  resetPasswordWithWrongEmail,
+  updatePasswordSuccessfully,
+  updateWithWrongPassword,
+  resetPassworWithWrongEmailFormat
 } from '../../../src/db/seeders/user';
+// import {
```

Comment:
Hi, Please take out this commented texts

Reply:
ok.